### DISCUSSION TITLE
Upload field for form builder

### DISCUSSION THREAD

**Discussion by martinmildner** [awcodes/filament-curator#144] [Ideas]
Just an idea: An upload field for the form builder that also adds a record to the media table.

**Comment by awcodes** [comment]
Hmm. Would have to give that some thought. Not sure how the internals would need to be for that to work.

**Comment by martinmildner** [comment]
In a project where users are allowed to upload images, but the admins should also be able to edit them in the backend, I am currently using the user updating event. Whenever a new image is uploaded via the form builder, a new media record is created.
